cobalt/devinfra/kokoro/bin/common.sh

```diff
@@ -216,7 +216,7 @@ upload_on_device_test_artifacts () {
   local out_dir=$4
 
   local artifact="${WORKSPACE_COBALT}/builder_tmp/${platform}_${config}/artifacts.tgz"
-  local gcs_archive_path="gs://$(get_project_name)-test-artifacts/${target_platform}/${KOKORO_GIT_COMMIT_src}/${KOKORO_ROOT_BUILD_ID}/${platform}_${config}/artifacts.tgz"
```

Comment:
I don't think we should change the variable referenced. We should instead add an export in `configure_environment` to workaround this weird behavior.

```
  if [ -z "${KOKORO_GIT_COMMIT_src+set}" ]; then
    export KOKORO_GIT_COMMIT_src=${KOKORO_GIT_COMMIT}
  fi
```